src/Dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -1,48 +1,55 @@
+import { useEffect, useState } from "react";
 import Header from "../Header/Header";
+import axios from "axios";
+import { useUser } from "../../../context/AuthContext";
+import DonutChart from "react-donut-chart";
 
 export default function Dasxboard() {
+  const { Token } = useUser();
+  const [todoCount, settodoCount] = useState<number>(0);
+  const [progressCount, setprogressCount] = useState<number>(0);
+  const [doneCount, setdoneCount] = useState<number>(0);
+  const [activatedEmployeeCount, setactivatedEmployeeCount] = useState<number>(0);
+  const [deactivatedEmployeeCount, setdeactivatedEmployeeCount] = useState<number>(0);
+
+  const getTasksCount = async () => {
+    try {
+      let response = await axios.get(
+        `https://upskilling-egypt.com:3003/api/v1/Task/count`,
```

Comment:
Having the URLs hardcoded like that is not a  good practice, consider storing the base URL as a constant and then use it every where.
You can also follow a similar structure like these [one](https://github.com/AnasKhaled7/PMS-React/blob/main/src/lib/APIs.ts) and store all application APIs one time and then use it when needed.